docs/recipes/code-coverage.md

```diff
@@ -85,11 +85,11 @@ Since it is unlikely you want `inline` source maps in your production code. You
 
 > WARNING: `BABEL_ENV=production` does not work on Windows, you must use the `set` keyword  (`set BABEL_ENV=production`).  For cross platform builds, check out [`cross-env`].
 
-Note that the build script really has very little to do with AVA, and is just a demonstration of how to use Babel's `env` configuration to manipulate your config so it's compatible with AVA.
+Note that the build script has very little to do with AVA and is just a demonstration of how to use Babel's `env` configuration to manipulate your config, so it's compatible with AVA.
 
 ### Use the Babel require hook
 
-To use the Babel require hook, add `babel-core/register` to the `require` section of you AVA config in `package.json`.
+To use the Babel `require` hook, add `babel-core/register` to the `require` section of you AVA config in `package.json`.
```

Comment:
[Babel just uses *require hook*](https://babeljs.io/docs/usage/babel-register/).

While we're updating this though, we should recommend using `babel-register`, not `babel-core/register` which is undocumented (and, without verifying, might very well be going away in Babel 7).